src/pages/calendar/components/TodoItem.jsx

```diff
@@ -95,6 +100,14 @@ export const TodoItem = ({ title, priority, assignedUser, id, ...rest }) => {
         visible={isEditVisible}
         setVisible={setEditVisible}
       />
+      <DeleteModal
+        visible={isDeleteModalOpen}
+        setVisible={setIsDeleteModalOpen}
+        deleteFn={() => deleteTask(id)}
```

Comment:
Please don't reduce anything in your code, ANYTHINK
onDelete - should be name of prop